### PR TITLE
Examples extended with switching Bootswatch theme

### PR DESCRIPTION
**Pull request by PanderMusubi**

### CODE REVIEW

**Comment by greyli** [comment]
What do you mean by "table fix"? Why add the "bug" label?

File: docs/index.rst [hunk 1]
```diff
@@ -67,6 +67,16 @@ Or run the full checks with tox:
...
 
 
+Upgrading
+---------
+
+The following upstream releases are used to upgrade Bootstrap Flask:
+- `Bootstrap <https://github.com/twbs/bootstrap/releases>`_
+- `Bootstrap Icons <https://github.com/twbs/icons/releases>`_
+- `Bootswatch <https://github.com/thomaspark/bootswatch/tags>`_
+- `Floating UI <https://github.com/floating-ui/floating-ui/releases>`_
```
Comment (greyli) [inline docs/index.rst:77]: What is it?

File: docs/index.rst [hunk 2]
```diff
@@ -67,6 +67,16 @@ Or run the full checks with tox:
     $ tox
 
 
+Upgrading
```
Comment (greyli) [inline docs/index.rst:70]: I don't think this section is necessary.

File: examples/bootstrap4/app.py [hunk 3]
```diff
@@ -89,6 +88,36 @@ class ContactForm(FlaskForm):
     im_accounts = FieldList(FormField(IMForm), min_entries=2)
 
 
+class BootswatchForm(FlaskForm):
+    """Form to test Bootswatch."""
+    #DO NOT EDIT! Use list-bootswatch.py te generate the Radiofield below.
```
Comment (greyli) [inline examples/bootstrap4/app.py:93]:
```suggestion
    # DO NOT EDIT! Use list-bootswatch.py to generate the Radiofield below.
```

File: examples/bootstrap5/app.py [hunk 4]
```diff
@@ -89,6 +88,40 @@ class ContactForm(FlaskForm):
     im_accounts = FieldList(FormField(IMForm), min_entries=2)
 
 
+class BootswatchForm(FlaskForm):
+    """Form to test Bootswatch."""
+    #DO NOT EDIT! Use list-bootswatch.py te generate the Radiofield below.
```
Comment (greyli) [inline examples/bootstrap5/app.py:93]:
```suggestion
    # DO NOT EDIT! Use list-bootswatch.py to generate the Radiofield below.
```

File: examples/bootstrap4/app.py [hunk 5]
```diff
@@ -89,6 +88,36 @@ class ContactForm(FlaskForm):
...
+                                 ('sandstone', 'Sandstone 4.6.1'),
+                                 ('simplex', 'Simplex 4.6.1'),
+                                 ('sketchy', 'Sketchy 4.6.1'),
+                                 ('slate', 'Slate 4.6.1'),
+                                 ('solar', 'Solar 4.6.1'),
+                                 ('spacelab', 'Spacelab 4.6.1'),
+                                 ('superhero', 'Superhero 4.6.1'),
+                                 ('united', 'United 4.6.1'),
+                                 ('yeti', 'Yeti 4.6.1'),
+                        ])
```
Comment (greyli) [inline examples/bootstrap4/app.py:117]: The indentation here is incorrect.

File: examples/bootstrap5/app.py [hunk 6]
```diff
@@ -89,6 +88,40 @@ class ContactForm(FlaskForm):
...
+                                 ('sketchy', 'Sketchy 5.1.3'),
+                                 ('slate', 'Slate 5.1.3'),
+                                 ('solar', 'Solar 5.1.3'),
+                                 ('spacelab', 'Spacelab 5.1.3'),
+                                 ('superhero', 'Superhero 5.1.3'),
+                                 ('united', 'United 5.1.3'),
+                                 ('vapor', 'Vapor 5.1.3'),
+                                 ('yeti', 'Yeti 5.1.3'),
+                                 ('zephyr', 'Zephyr 5.1.3'),
+                        ])
```
Comment (greyli) [inline examples/bootstrap5/app.py:121]: Same here.

File: examples/bootstrap5/app.py [hunk 7]
```diff
@@ -155,10 +188,26 @@ def test_nav():
     return render_template('nav.html')
 
 
+@app.route('/bootswatch', methods=['GET', 'POST'])
+def test_bootswatch():
+    form = BootswatchForm()
+    if form.validate_on_submit():
+        print('FFFFF', app.config['BOOTSTRAP_BOOTSWATCH_THEME'])
```
Comment (greyli) [inline examples/bootstrap5/app.py:195]: The print statement needs to be removed.

File: examples/bootstrap4/app.py [hunk 8]
```diff
@@ -155,10 +184,26 @@ def test_nav():
     return render_template('nav.html')
 
 
+@app.route('/bootswatch', methods=['GET', 'POST'])
+def test_bootswatch():
+    form = BootswatchForm()
+    if form.validate_on_submit():
+        print('FFFFF', app.config['BOOTSTRAP_BOOTSWATCH_THEME'])
```
Comment (greyli) [inline examples/bootstrap4/app.py:191]: Remove the print statement.

File: examples/bootstrap4/app.py [hunk 9]
```diff
@@ -89,6 +88,36 @@ class ContactForm(FlaskForm):
     im_accounts = FieldList(FormField(IMForm), min_entries=2)
 
 
+class BootswatchForm(FlaskForm):
+    """Form to test Bootswatch."""
+    #DO NOT EDIT! Use list-bootswatch.py te generate the Radiofield below.
+    render = RadioField(default='default',
```
Comment (greyli) [inline examples/bootstrap4/app.py:94]: `theme` or `theme_name` will be a better name than `render`.

**Comment by PanderMusubi** [comment]
> What do you mean by "table fix"? Why add the "bug" label?

Ah, the big fix was regarding pagination, not table. Bug label because it was a bug, it broke the examples.

File: examples/list-bootswatch.py [hunk 10]
```diff
@@ -0,0 +1,21 @@
+'''List Bootstrap themes RadioField.'''
```
Comment (greyli) [inline examples/list-bootswatch.py:1]: Use double quotes for docstring.

File: examples/list-bootswatch.py [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+
+from os import listdir
+
+def list(version):
+    '''List template file names.'''
```
Comment (greyli) [inline examples/list-bootswatch.py:6]: Double quotes.

File: examples/list-bootswatch.py [hunk 12]
```diff
@@ -0,0 +1,21 @@
+'''List Bootstrap themes RadioField.'''
+
+from os import listdir
+
```
Comment (greyli) [inline examples/list-bootswatch.py:4]: Two empty lines

File: examples/list-bootswatch.py [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+    base = f'../flask_bootstrap/static/bootstrap{version}/css/bootswatch'
+    name = ''
+    for directory in sorted(listdir(base)):
+        with open(f'{base}/{directory}/_bootswatch.scss') as scss:
+            for line in scss:
+                name = line.strip()[3:]
+                break
+        print(f"                                 ('{directory}', '{name}'),")
+    print("                        ])")
+
```
Comment (greyli) [inline examples/list-bootswatch.py:19]: Two empty lines.

File: examples/update-icons.py [hunk 14]
```diff
@@ -13,7 +13,7 @@ def parse(filename):
 
 def generate(version):
     '''Write the HTML template file.'''
```
Comment (greyli) [inline examples/update-icons.py:15]: Double quotes.

**Review by greyli — CHANGES_REQUESTED** [review]

File: docs/index.rst (hunk 1 above)
Comment (PanderMusubi) [reply docs/index.rst:77]: This is for maintainers that want to do an upgrade to quickly see where to get the latest version to upgrade of what is shipped with Bootstrap-Flask.

File: examples/bootstrap4/app.py (hunk 3 above)
Comment (PanderMusubi) [reply examples/bootstrap4/app.py:93]: Fixed

File: examples/bootstrap5/app.py (hunk 4 above)
Comment (PanderMusubi) [reply examples/bootstrap5/app.py:93]: Fixed

File: examples/list-bootswatch.py (hunk 10 above)
Comment (PanderMusubi) [reply examples/list-bootswatch.py:1]: Fixed

File: examples/list-bootswatch.py (hunk 11 above)
Comment (PanderMusubi) [reply examples/list-bootswatch.py:6]: Fixed

File: examples/list-bootswatch.py (hunk 13 above)
Comment (PanderMusubi) [reply examples/list-bootswatch.py:19]: Fixed

File: examples/update-icons.py (hunk 14 above)
Comment (PanderMusubi) [reply examples/update-icons.py:15]: Fixed

File: examples/bootstrap4/app.py (hunk 5 above)
Comment (PanderMusubi) [reply examples/bootstrap4/app.py:117]: What specifically?

File: docs/index.rst (hunk 2 above)
Comment (PanderMusubi) [reply docs/index.rst:70]: See comment above on the list of upstream repo release URLs

**Comment by greyli** [comment]
For unrelated features or bugs, please submit a separate PR. The #257 also contains the same type of fix, please create a new PR for changes like this.

File: docs/index.rst (hunk 1 above)
Comment (greyli) [reply docs/index.rst:77]: The doc is for users. You can create a wiki page for content like this.

File: examples/bootstrap4/app.py (hunk 5 above)
Comment (greyli) [reply examples/bootstrap4/app.py:117]:
```
    render = RadioField(
        default='default',
        choices=[
            ('default', 'none'),
            ('cerulean', 'Cerulean 4.6.1'),
            ('cosmo', 'Cosmo 4.6.1'),
            ('cyborg', 'Cyborg 4.6.1'),
            ('darkly', 'Darkly 4.6.1'),
            ('flatly', 'Flatly 4.6.1'),
            ('journal', 'Journal 4.6.1'),
            ('litera', 'Litera 4.6.1'),
            ('lumen', 'Lumen 4.6.1'),
            ('lux', 'Lux 4.6.1'),
            ('materia', 'Materia 4.6.1'),
            ('minty', 'Minty 4.6.1'),
            ('pulse', 'Pulse 4.6.1'),
            ('sandstone', 'Sandstone 4.6.1'),
            ('simplex', 'Simplex 4.6.1'),
            ('sketchy', 'Sketchy 4.6.1'),
            ('slate', 'Slate 4.6.1'),
            ('solar', 'Solar 4.6.1'),
            ('spacelab', 'Spacelab 4.6.1'),
            ('superhero', 'Superhero 4.6.1'),
            ('united', 'United 4.6.1'),
            ('yeti', 'Yeti 4.6.1'),
        ]
    )
```

File: docs/index.rst (hunk 1 above)
Comment (PanderMusubi) [reply docs/index.rst]: Rmoved to add to wiki

File: docs/index.rst (hunk 2 above)
Comment (PanderMusubi) [reply docs/index.rst]: Removed to add to wiki

File: examples/bootstrap4/app.py (hunk 5 above)
Comment (PanderMusubi) [reply examples/bootstrap4/app.py]: Fixed

File: examples/bootstrap4/app.py (hunk 8 above)
Comment (greyli) [reply examples/bootstrap4/app.py:194]: ?

Comment (PanderMusubi) [reply examples/bootstrap4/app.py]: Fixed, was some debug remnant.

File: examples/bootstrap5/app.py (hunk 7 above)
Comment (PanderMusubi) [reply examples/bootstrap5/app.py]: Fixed, was some debug remnant.

File: examples/bootstrap5/app.py (hunk 6 above)
Comment (PanderMusubi) [reply examples/bootstrap5/app.py]: Meanwhile fixed

File: examples/bootstrap4/app.py (hunk 9 above)
Comment (PanderMusubi) [reply examples/bootstrap4/app.py]: fixed

**Review by greyli — APPROVED** [review]

**Pull request merged**